Fix for grammar and typos

Bunfus opened the pull request:
# About the pull request
This PR doesn't add anything new, but it does fix the majority of the grammar and formatting issues across all files (Fixes #11009).

# Explain why it's good for the game
Having less typos is good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots</summary>

</details>

# Changelog
:cl: Nemesis1337
spellcheck: fixed majority of the typos in code comments
spellcheck: fixed majority of the typos in examine text
code: fixed some typos in var names
/:cl:

Bunfus committed:
Fixed some table issues

Bunfus committed:
Fixed gambling table

Bunfus committed:
Fix many typos

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'barricade-tweaks' of https://github.com/Bunfus/cmss13 into barricade-tweaks

Bunfus committed:
Fixed typos

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Fixed path

Bunfus committed:
Merge branch 'master' into barricade-tweaks

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Var edits end fix

Bunfus committed:
Extra typo fixes

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Puckaboo2 commented:
Yeah, you're updating way too many files at once. It's best you split up the changes as I have and start with changes people can actually see in-game, like descriptions.

Bunfus committed:
More grammar fixes

Bunfus committed:
More grammar fixes

Bunfus committed:
Merge branch 'master' into barricade-tweaks

Bunfus committed:
Missed a few

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Removed table PR and map changes

Bunfus committed:
Missed a few

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Fixup maps in TGM format

066ceb592003: maps/map_files/FOP_v2_Cellblocks/Prison_Station_FOP.dmm
066ceb592003: maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm
066ceb592003: maps/map_files/LV624/standalone/clfship.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Actually there was more

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge branch 'master' into barricade-tweaks

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

Bunfus committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into grammar-tweaks

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into grammar-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into grammar-tweaks

Bunfus committed:
Merge branch 'master' into barricade-tweaks

Detective-Google commented:
this has been open some months without much interaction, and this is realistically never going to be reviewed at the scale it encompasses, thank you for the effort! 

It's recommended to focus on fixing typos with especially high visibility or those that intersect a project you are already working on, instead of shotgun blasting huge amounts of grammar edits at once.
